src/components/ImagePreview.tsx

```diff
@@ -40,7 +40,7 @@ const ImagePreview = (props: Props): JSX.Element | null => {
       <figure className={styles.preview}>
         {document.extension === 'jpeg' || document.extension === 'png' ? (
           <img src={downloadUrl} alt={documentSubmission.documentType.title} />
-        ) : document.extension === 'heic' ? (
+        ) : document.extension === 'heic' || document.extension === 'heif' ? (
```

Comment:
We're starting to see a pattern here for files with extensions that need to be converted. Might be worth considering adding them in an array and then check if the current file extension was found in that array.

Reply:
I know it's not part of this PR but also noticed that the `heicImage` variable name might be misleading as the `src` being passed to the `<img>` in line 50 will sometimes have a `heif` extension.

Reply:
Refactored the code to allow document extension to be checked against an array of extensions that requires conversion. 
This also includes an additional change to check if the document with a certain type of extension does not require conversion.